dts/msm8226/apq8026-samsung-r04.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0xC708FF01 4 0x20000>;
```

Comment:
did you arbitrarily pick 4 here or the device doesn't boot with any other value?

the -rXX in the name reflects the second cell in this property (4). Usually you would pick the highest number available in the downstream sources or the dtbs dumped from the device itself.